cypress/integration/seededFlows/loggedOutFlows/namespacedForemMobile.spec.js

```diff
@@ -0,0 +1,109 @@
+describe('Namespaced ForemMobile functions', () => {
+  function waitForMobileToBootstrap() {
+    cy.get('body').should('have.attr', 'data-runtime', 'ForemWebView-iOS');
```

Comment:
Wait on this attribute so that `win.ForemMobile` will exist in tests.

Reply:
Since we're introducing [waitOnBaseData](https://github.com/forem/forem/pull/15212/files#diff-a082293aa8866316df6c6168cd27fb1cec6ce34b6d285865e27a1ceaa95a1462R1) it seems like it would be better to check for that one. I'll even consider moving this out as a re-usable function (like `loginUser`) if I need to use it again in another cypress test.

This trick however, never crossed my mind but makes absolute sense (give Cypress the responsibility to wait until the page has loaded). Thanks for this!

Reply:
`waitForMobileToBootstrap` is a function only for Cypress. `waitOnBaseData` is application code. Waiting on the `<body />` tag's `data-loaded="true"` isn't necessary for end-to-end (E2E) test as the tests are driven by what a user does and sees, not the internals of how a user's info is loaded. 

`waitForMobileToBootstrap` is checking implementation, but for testing that the bridge for web/mobile exists. It's an exception in terms of E2E tests.